dcspy/logitech.py

```diff
@@ -49,7 +49,6 @@ def __init__(self, parser: ProtocolParser, sock: socket, **kwargs) -> None:
         self.plane_name = ''
         self.bios_name = ''
         self.plane_detected = False
-        self.lcdbutton_pressed = False
```

Comment:
This is still referenced in the broken test_logitech.test_keyboard_check_buttons` unit test so needs to be removed from there as well once its figured out what that test should be doing.